sandy/core/endf6.py

```diff
@@ -15,6 +15,7 @@
 
 
 import pandas as pd
+import numpy
```

Comment:
we usually import it as 
```
import numpy as np
```

Reply:
why I don't see `import numpy as np`?

Anyway, do not import numpy since you don't use it